contracts/peg/SimpleFeiDaiPSM.sol

```diff
@@ -36,7 +36,6 @@ contract SimpleFeiDaiPSM {
         uint256 minAmountOut
```

Comment:
```suggestion
        uint256 // minAmountOut
```

Reply:
I forget whether the syntax is to omit the uint256 or not when having unnamed variables. This should be done below too

Reply:
syntax is to omit the uint256 variable name only, not the type (that's used by the compiler to know this function still expects an uint256, but just silences the "variable unused" warning)

Reply:
Omitted the variable name, kept the type